cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/DefaultApplicationSpecification.java

```diff
@@ -68,6 +70,38 @@ public DefaultApplicationSpecification(String name, String description, String c
                                          Map<String, WorkerSpecification> workers,
                                          Map<String, Plugin> plugins) {
     this.name = name;
+    this.appVersion = ApplicationId.DEFAULT_VERSION;
```

Comment:
can we do just ``this(name, ApplicationId.DEFAULT_VERSION, description, ......)``